fabric-resource-loader-v0/src/main/resources/assets/fabric-resource-loader-v0/lang/en_us.json

```diff
@@ -1,4 +1,5 @@
 {
+  "pack.description.fabricApi": "Mod resources.",
```

Comment:
Can we use another id maybe? How about `pack.description.modResources`?

Reply:
I don't know, I shall let other FAPI developers choose that.

Reply:
I am one of them. :smile: